### PR TITLE
Fix the link to `Monolog\Processor\ProcessorInterface` in processors.rst

### PR DESCRIPTION
**Pull request by marcinowski**
Bug appeared in 4.2 
Fixes the #10795

### CODE REVIEW

**Comment by marcinowski** [comment]
I can agree with that. Since there's absolutely no trace of `ProcessorInterface` neither in symfony/symfony, symfony/monolog-bundle or symfony/monolog-bridge. it seems like the solution would be to completely remove this whole paragraph. 
The current faulty Monolog reference comes from #10483, which simply replaced the reference.

**Comment by xabbuh** [comment]
indeed, we need to wait for symfony/monolog-bundle#285 to be able to document this feature again

File: logging/processors.rst [hunk 1]
```diff
@@ -298,3 +246,5 @@ the ``monolog.processor`` tag:
         $container
             ->register(SessionRequestProcessor::class)
             ->addTag('monolog.processor', array('channel' => 'main'));
+
+.. _ProcessorInterface: https://github.com/Seldaek/monolog/blob/master/src/Monolog/Processor/ProcessorInterface.php
```
Comment (xabbuh) [inline logging/processors.rst]: can be removed too

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @marcinowski.

**Pull request closed without merging**